src/interpreter/ByteCode.cpp

```diff
@@ -0,0 +1,71 @@
+/*
+ * Copyright (c) 2022-present Samsung Electronics Co., Ltd
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#include "Walrus.h"
+
+#include "interpreter/ByteCode.h"
+#include "runtime/ObjectType.h"
+
+namespace Walrus {
+
+// Return static size of opcodes
+size_t ByteCode::getBaseSize()
```

Comment:
The `getBaseSize` should return `g_byteCodeInfo[opcode].m_size;`. It could be a static function with an opcode argument.

Reply:
It is changed to be a static function now.